tests/test_files.py

```diff
@@ -63,12 +63,6 @@ def test_update_file(celery_app, client, annex_path):
     response = client.simulate_post(
         '/datasets/{}/files/LICENSE'.format(ds_id), body=file_data)
     assert response.status == falcon.HTTP_OK
-    # Then update it
-    file_data = 'New test LICENSE'
-    response = client.simulate_put(
-        '/datasets/{}/files/LICENSE'.format(ds_id), body=file_data)
-    assert response.status == falcon.HTTP_OK
-    # Load the dataset to check for the updated file
```

Comment:
The test should still test updating a file, this can be simluate_post. If this change is being made, we should remove the PUT implementation, otherwise it's useful to keep it covered by this test.

Reply:
Updated and working. It appears they anticipate the possibility of a sixth release candidate, so perhaps merging this should be put off (unless you'd suggest otherwise).